bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/configurable/ConfigResolver.java

```diff
@@ -85,6 +85,13 @@ public ConfigResolver(Module rootModule, Map<Module, VariableKey[]> configVarMap
                 configValue.ifPresent(o -> configValueMap.put(varKey, o));
             }
         }
+        for (ConfigProvider provider : runtimeConfigProviders) {
+            try {
+                provider.complete();
+            } catch (ConfigException e) {
+                diagnosticLog.warn(e.getErrorCode(), e.getMessage(), e.getArgs());
```

Comment:
are we relaying on the throw instead of the log? will this work if there are multiple variables ?